src/backend/joanie/core/models/products.py

```diff
@@ -379,6 +381,28 @@ class OrderGroup(BaseModel):
         on_delete=models.CASCADE,
     )
     is_active = models.BooleanField(_("is active"), default=True)
+    # Available start to end period of activation of the OrderGroup
+    start = models.DateTimeField(
+        help_text=_("order group’s start date and time"),
+        verbose_name=_("order group start datetime"),
+        blank=True,
+        null=True,
+    )
+    end = models.DateTimeField(
+        help_text=_("order group’s end date and time"),
```

Comment:
This comment applies here also
https://github.com/openfun/joanie/pull/1032/files#r1946599654